pkg/kubectl/cmd/config/flags.go

```diff
@@ -45,15 +47,15 @@ func (f *kubectlConfigPrintFlags) ToPrinter() (printers.ResourcePrinter, error)
 	}
 
 	if p, err := f.JSONYamlPrintFlags.ToPrinter(outputFormat); !printers.IsNoCompatiblePrinterError(err) {
-		return p, err
+		return f.TypeSetter.WrapToPrinter(p, err)
```

Comment:
would there need to be a nil check here?

Reply:
> would there need to be a nil check here?

No.  nil receivers are allowed and the wrap handles that case.